src/gtc/passes/oir_optimizations/horizontal_execution_merging.py

```diff
@@ -18,9 +18,94 @@
 from typing import Any, Callable, Dict, List, Optional, Set, Tuple, Union
 
 from eve import NodeTranslator, SymbolTableTrait
+from gt4py.definitions import Extent
```

Comment:
It's unfortunate that we keep needing/using this type in GTC. Would be good to port over into a separate utility to break the circular dependency (though it's read-only).

Reply:
I added an issue to track this #592.